test/unit/test_utils.c

```diff
@@ -437,13 +437,35 @@ START_TEST(test_as_key_move_success)
 	src->valuep = &src->value;
 
 	ck_assert(as_key_move(dst, src));
-	ck_assert(dst->value.integer.value == src->value.integer.value);
+	ck_assert_int_eq(dst->value.integer.value, src->value.integer.value);
 	cf_free(test_key_value);
 	cf_free(dst);
 	cf_free(src);
 }
 END_TEST
 
+START_TEST(test_secondary_index_param_with_ctx)
```

Comment:
add a unit test to sindex record without "context" in server response (index->ctx be null)